src/Interpreters/executeQuery.cpp

```diff
@@ -301,28 +301,16 @@ static void onExceptionBeforeStart(const String & query_for_logging, ContextPtr
         span.operation_name = "query";
         span.start_time_us = current_time_us;
         span.finish_time_us = time_in_microseconds(std::chrono::system_clock::now());
-
-        /// Keep values synchronized to type enum in QueryLogElement::createBlock.
-        span.attribute_names.push_back("clickhouse.query_status");
-        span.attribute_values.push_back("ExceptionBeforeStart");
-
-        span.attribute_names.push_back("db.statement");
-        span.attribute_values.push_back(elem.query);
-
-        span.attribute_names.push_back("clickhouse.query_id");
-        span.attribute_values.push_back(elem.client_info.current_query_id);
-
-        span.attribute_names.push_back("clickhouse.exception");
-        span.attribute_values.push_back(elem.exception);
-
-        span.attribute_names.push_back("clickhouse.exception_code");
-        span.attribute_values.push_back(elem.exception_code);
-
+        span.attributes.reserve(7);
+        span.attributes.push_back(Tuple{"clickhouse.query_status", "ExceptionBeforeStart"});
+        span.attributes.push_back(Tuple{"db.statement", "elem.query"});
```

Comment:
I think you want to write not `"elem.query"`, but `elem.query`. There is the same mistake in other lines.